package.json

```diff
@@ -43,7 +43,8 @@
     "ddev:composer-update": "ddev composer update && ddev composer update --lock",
     "ddev:unit-tests": "ddev exec phpunit",
     "ddev:e2e-tests": "cp -n .env.e2e.example .env.e2e && ddev php tests/e2e/PHPUnit/setup.php && ddev exec phpunit -c tests/e2e/phpunit.xml.dist",
-    "ddev:test": "yarn run ddev:unit-tests && yarn run ddev:e2e-tests",
+    "ddev:pw-tests": "npx playwright test --grep @ci",
```

Comment:
I would also add e.g. `"ddev:pw-tests-all": "npx playwright test --hosted",`

Oh, and can it be `ddev run npx .....`? To avoid installing on host.